decidim-proposals/spec/forms/decidim/proposals/admin/admin_proposal_form_spec.rb

```diff
@@ -6,8 +6,8 @@ module Decidim
   module Proposals
     module Admin
       describe ProposalForm do
-        it_behaves_like "a proposal form", skip_etiquette_validation: true, i18n: true, admin: true
-        it_behaves_like "a proposal form with meeting as author", skip_etiquette_validation: true, i18n: true, admin: true
+        it_behaves_like "a proposal form", i18n: true, admin: true
```

Comment:
I think we can drop `skip_etiquette_validation` altogether at decidim-proposals/spec/shared/proposal_form_examples.rb, as it isn't being used anymore. Can you remove that?

Reply:
fixed in 3f85a9e8a8